src/test/java/org/takes/misc/HrefTest.java

```diff
@@ -37,7 +38,12 @@
 
     /**
      * Href can build an URI.
+     * @todo #607:30min Href query param names are stored as Map keys,
```

Comment:
@Happy-Neko why not just use the correct map type in `Href` instead of adding a puzzle? Seems to me it takes more time to write this puzzle than to change the map type.